.php_cs.dist

```diff
@@ -19,5 +19,12 @@ return PhpCsFixer\Config::create()
         ],
         'no_empty_phpdoc' => true,
         'no_superfluous_phpdoc_tags' => true,
+        'visibility_required' => [
+            'elements' => [
+                'const',
```

Comment:
Hmm, not sure if https://github.com/php-http/boilerplate/blob/948e25ed69c27be69174ebb3bd0bcc11ecbc545f/composer.json#L14 make sense, then.

What do you think?

Reply:
we built this back when we started with php 5.5 - i guess the boilerplate should say 7.1, as if we start anything new or update that stuff in any project, we should start with php 7.1